CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## vNext (TBD)
 
 ### Enhancements
-* None
+- Added 3 new methods on lists ([#3324](https://github.com/realm/realm-js/issues/3324)):
+  - `list.remove(index)`: removes the element of the list at the specified index.
+  - `list.move(from, to)`: moves one element of the list from one index to another.
+  - `list.swap(index1, index2)`: swaps the positions of the elements of the list at two indexes.
```

Comment:
```suggestion
* Added 3 new methods on lists ([#3324](https://github.com/realm/realm-js/issues/3324)):
  * `list.remove(index)`: removes the element of the list at the specified index.
  * `list.move(from, to)`: moves one element of the list from one index to another.
  * `list.swap(index1, index2)`: swaps the positions of the elements of the list at two indexes.
```

Reply:
Just to be consistant.  I think at some point we landed on "*" for bullets in the changelog.